test/test_playbooks/auth_source_ldap.yml

```diff
@@ -0,0 +1,29 @@
+---
+- hosts: tests
+  gather_facts: false
+  tasks:
+  - name: Clear API cache
+    file:
+      name: ~/.cache/apypie
+      state: absent
```

Comment:
Well, i am sorry, this did not fix the problem. The request to download the api.json is still not in the fixture.
The reordering in test_crud did it.

Reply:
Makes sense (since it's not in any other playbook). Removing it.

Reply:
That is not my point. It just didn't work that way. I really want the tests to be able to run independently. And to accomplish that, we must invalidate/clear the cache after setting up the fixtures and before the first test call is recorded.

Reply:
Ok, but do you want me to keep the task or remove it?

Reply:
It's OK without it for the moment (i think, you need to rerecord however).
We need to find a general solution to this.
#280